frontend/src/App.js

```diff
@@ -1,25 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+import {Route, Switch, BrowserRouter as Router} from 'react-router-dom';
+import AppHeader from "./components/AppHeader";
+import SearchProducts from "./components/SearchProducts";
+import {useEffect, useState} from "react";
+import {getUserProductsList} from "./service/ApiService";
+import ProductList from "./components/ProductList";
+
+
+
 
 function App() {
+    const [products, setProducts] = useState([])
+
+    useEffect(() => {
+        getUserProductsList()
+            .then(setProducts)
+            .catch((error) => console.error(error))
+    }, [])
+
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+      <Router>
+          <AppHeader />
+          <Switch>
+              <Route exact path="/">
+                  <SearchProducts setProducts={setProducts} products={products}/>
+                  <ProductList products={products}/>
+              </Route>
+              <Route path="/productDetails">
+              </Route>
```

Comment:
this route currently does not include a component and would lead the user to a non-existing empty page - did you maybe forget to include your component here?

Reply:
I have already laid out the route for myself. My next step